web/components/TextBlock/Tickets/Tickets.tsx

```diff
@@ -4,6 +4,10 @@ import checkmarkIcon from '../../../images/checkmark.svg';
 import crossIcon from '../../../images/cross.svg';
 import styles from './Tickets.module.css';
 import clsx from 'clsx';
+import { Fragment } from 'react';
+import Block from '../Block';
```

Comment:
I don't know why the linter doesn't flag this as an unused import, but it seems to be unused now…

```suggestion
```